toolkit/tools/imagecustomizer/README.md

```diff
@@ -44,7 +44,9 @@ Disadvantages:
    For documentation on the supported configuration options, see:
    [Mariner Image Customizer configuration](./docs/configuration.md)
 
-3. Run the Mariner Image Customizer tool.
+3. Install prerequisites required to run Mariner Image Customizer: `qemu-img,rpm,dd,lsblk,losetup,sfdisk,udevadm,flock,blkid,openssl,sed,createrepo,squashfs-tools,genisoimage`
```

Comment:
nit: Install prerequisites for running Mariner Image Customizer
prerequisites required is redundant...